src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -1067,7 +1068,8 @@ void CacheManager::ResetIgnitionCycles() {
 
 int CacheManager::TimeoutResponse() {
   CACHE_MANAGER_CHECK(0);
-  return pt_->policy_table.module_config.timeout_after_x_seconds;
+  return pt_->policy_table.module_config.timeout_after_x_seconds *
```

Comment:
@dev-gh TimeoutResponse used in multiple places and in a lot of Unit tests. Decreasing accuracy of time measuring on interface level make some unit tests (that have to wait asynchronous operations) too long ( 1 second minimum). I think that we should use milliseconds in SDL everywhere to avoid a lot of convertions (out timer works on milliseconds) and if in database or stored json or update from mobile, we get seconds, we should convert and store it as milliseconds as soon as possible.